docs/deployment/configuring_logging.md

```diff
@@ -74,3 +74,22 @@ logging:
   loki:
     enabled: false
 ```
+
+# Configuring central Loki storage capacity
+
+By default, the central Loki has 100Gi of storage capacity.
+To overwrite the current central Loki storage capacity, the `logging.loki.garden.storage` setting in the gardenlet's component configuration should be altered.
+If you need to increase it you can do so without loosing the current data by specifying higher capacity. Doing so, the Loki's `PersistentVolume` capacity will be increased instead of deleting the current PV.
```

Comment:
```suggestion
If you need to increase it you can do so without losing the current data by specifying higher capacity. Doing so, the Loki's `PersistentVolume` capacity will be increased instead of deleting the current PV.
```